cordformation/src/test/kotlin/net/corda/plugins/CordformTest.kt

```diff
@@ -65,7 +65,7 @@ class CordformTest {
         createBuildFile(buildFileResourceName)
         return GradleRunner.create()
                 .withProjectDir(testProjectDir.root)
-                .withArguments("deployNodes", "-s", "--info", "-g", testGradleUserHome)
+                .withArguments("deployNodes", "-s", "--getInfo", "-g", testGradleUserHome)
```

Comment:
I think this must be a `find/replace` mistake :wink:

Reply:
I should stop trusting intellij to rename things :) thanks @chrisr3 !